urbanopt/geojson-modelica-translator
# Add Level 1 5G DES system

#### Any background context you want to provide?
The 5G models were buried in the "level 3" type of GMT configuration with limited interconnections to other components. 

#### What does this PR accomplish?
This creates the 5G system with waste heat and GHX from a system parameter file and set of loads (ideally exported from OpenStudio). The 5G model is the series model that is in the MBL. It was imported into the GMT and templatized to replace the list of building loads.

#### How should this be manually tested?
* generate the 5G system `poetry run pytest ./tests/GMT_Lib/test_gmt_lib_des.py`
* Load the resulting package into Dymola and simulate. Package will be in the `./tests/GMT_Lib/output/DES_5G` directory.
* Rename SystemParameters.data to SystemParameters.param_template. There were two places were the instance data were being stored and it caused issues.

Note that none of the parameters are exposed and the partial model contains many of the setpoints for the waste heat and GHX. Those variables should be exposed to allow easy "setting".

#### What are the relevant tickets?

#### Screenshots (if appropriate)
